packages/react/src/components/ColorPicker/ColorPicker.test.tsx

```diff
@@ -278,53 +282,59 @@ describe('ColorPicker', () => {
     expect(colorChangeSpy).toHaveBeenCalledTimes(0);
 
     // Tab between text inputs checking state after each time.
-    const allInputs = wrapper.find(`.${inputClassName} input`);
-    expect(allInputs.length).toBe(5);
+    const inputs = getAllByRole('textbox') as HTMLInputElement[];
+    expect(inputs.length).toBe(5);
 
-    allInputs.forEach(input => {
-      input.simulate('focus');
-      input.simulate('blur');
+    inputs.forEach(input => {
+      userEvent.tab();
+      userEvent.tab({ shift: true });
 
       expect(colorPicker!.color.hex).toEqual(colorStringValue);
       expect(colorChangeSpy).toHaveBeenCalledTimes(0);
     });
   });
 
   it('allows updating text fields', () => {
-    wrapper = mount(<ColorPicker onChange={onChange} color="#000000" componentRef={colorPickerRef} />);
+    const { container, getAllByRole } = render(
+      <ColorPicker onChange={onChange} color="#000000" componentRef={colorPickerRef} />,
+    );
 
-    const inputs = wrapper.getDOMNode().querySelectorAll('.ms-ColorPicker-input input') as NodeListOf<HTMLInputElement>;
+    const inputs = getAllByRole('textbox') as HTMLInputElement[];
 
     const redInput = inputs[1];
-    ReactTestUtils.Simulate.input(redInput, mockEvent('255'));
+    userEvent.clear(redInput);
+    userEvent.paste(redInput, '255');
```

Comment:
Did typing it not work? I think that would be closer to the intent of the test, even if you have to change the number of onChange calls.

Reply:
Just made the change to `userEvent.type` with updated onChange calls. Kept some instances of `userEvent.paste` where it made sense though